packages/styles/scss/components/_hero.scss

```diff
@@ -30,19 +30,21 @@
   width: 100%;
   display: grid;
   position: relative;
-  grid-template-rows:
-    minmax($spacing-hero-image-max-height, 1fr)
-    $spacing-hero-tooltip-height
-    $spacing-hero-card-corner-cut-height-sm
-    auto;
   grid-template-columns: 1fr;
   overflow: hidden;
+  @include hero-grid-rows(1rem);
+
+  &--image {
+    display: grid;
```

Comment:
Does the image itself really need this grid?

Reply:
It does, Without it the styling gets off.